services/movie_session.py

```diff
@@ -42,3 +42,15 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    return [
+        {
+            "row": session["ticket__row"],
+            "seat": session["ticket__seat"]
+        }
+        for session in MovieSession.objects.filter(
+            ticket__movie_session=movie_session_id
+        ).values("ticket__row", "ticket__seat")
```

Comment:
it's fine, as for me, but you could also create additional variable to increase readability